src/features/unified-share-modal/messages.js

```diff
@@ -171,12 +171,22 @@ const messages = defineMessages({
         description: 'Label for confirmation modal to remove a shared link (title-case)',
         id: 'boxui.unifiedShare.removeLinkConfirmationTitle',
     },
+    removeCollaboratorConfirmationTitle: {
+        defaultMessage: 'Remove Collaborator',
+        description: 'Label for confirmation modal to remove a collaborator (title-case)',
+        id: 'boxui.unifiedShare.removeCollaboratorConfirmationTitle',
+    },
     removeLinkConfirmationDescription: {
         defaultMessage:
             'This will permanently remove the shared link. If this item is embedded on other sites it will also become inaccessible. Any custom properties, settings and expirations will be removed as well. Do you want to continue?',
         description: 'Description for confirmation modal to remove a shared link',
         id: 'boxui.unifiedShare.removeLinkConfirmationDescription',
     },
+    removeCollaboratorConfirmationDescription: {
+        defaultMessage: 'Are you sure you want to remove {name} as a collaborator on this App?',
```

Comment:
```suggestion
        defaultMessage: 'Are you sure you want to remove {name} as a collaborator?',
```
USM can be used on anything that can have a collaborator

Reply:
im seeing these messages in another message file you updated above, why do we need both of them?

Reply:
I updated the message leaving only `Are you sure you want to remove {name} as a collaborator?`, thanks!

Yeah, probably I don't need to have them in both files. Added them to both files as seen the title and description for removing link modal in both places.

Looks like it works fine with one file, so I removed the duplicate.